lib/pre-commit/template.rb

```diff
@@ -0,0 +1,83 @@
+require "date"
+require "erb"
+require "fileutils"
+
+module PreCommit
+  class Template
+    TEMPLATE_DIR = File.expand_path("../../../templates/gem", __FILE__)
```

Comment:
We might want to call this `templates/plugin`. Not a big deal though, this is fine.